Articles/WhatIs/WhatIsOnlineLearning.md

```diff
@@ -1,18 +1,28 @@
+<!--- body start --->
 
-#### Contributed by [David Bernholdt](https://github.com/bernhold)
+In the realm of high-performance computing (HPC) and scientific computing, online learning resources span a broad range and have become indispensable in enabling practitioners and researchers to stay abreast of the rapidly evolving technological landscape. These resources offer access to cutting-edge tools and techniques and aid in building the skills and knowledge required to efficiently leverage HPC systems and tackle intricate scientific problems.
 
+Universities and specialized online learning companies provide formal, structured and comprehensive courses that cover a broad spectrum of topics, from fundamental programming and data analysis to advanced parallel programming. These courses often provide certifications or credentials that can aid in career growth or demonstrate competence.
+
+Less formal and more specialized training materials offered by supercomputing centers, computing facilities, individual educators, and enthusiasts can be especially valuable in addressing specific challenges or knowledge gaps. Such resources include tutorials, blogs, documentation of various types,  as well as multimedia content such as webinars, video lectures, full sets of video+audio+slides, or full interactive sessions with screen sharing.
+
+It should be noted that while the majority of content is available asynchronously (i.e., recorded), live training events that are available for remote participation have become increasingly common. The availability of both asynchronous and live online training events is crucial in providing flexibility and accessibility for learners. Asynchronous resources permit learners to study at their own pace, while live online training events provide real-time interaction with instructors and other learners. Other than the convenience of allowing learners to access materials from anywhere with an internet connection, online learning resources can be made available 24/7, allowing learners to access them whenever it suits their schedule and often in a format that best suits their learning style. They are most times more cost-effective than in-person training and when done efficiently can provide opportunities for learners to interact with other students and instructors through discussion forums, chat rooms, and other online tools. This can enhance the learning experience and provide a sense of community for learners.
```

Comment:
```suggestion
It should be noted that while the majority of content is available asynchronously (i.e., recorded), live training events that are available for remote participation have become increasingly common. The availability of both asynchronous and live online training events is crucial in providing flexibility and accessibility for learners. Asynchronous resources permit learners to study at their own pace, while live online training events provide real-time interaction with instructors and other learners. In addition to the convenience of allowing learners to access materials from anywhere with an internet connection, online learning resources can be made available 24/7, allowing learners to access them whenever it suits their schedule and often in a format that best suits their learning style. These online resources are most often more cost-effective than in-person training, and when done efficiently can provide opportunities for learners to interact with other students and instructors through discussion forums, chat rooms, and other online tools. This can enhance the learning experience and provide a sense of community for learners.
```